### DISCUSSION TITLE
How does this compare with Parallels' Rosetta x64 execution on macOS Apple Silicon systems?

### DISCUSSION THREAD

**Discussion by ylluminate** [ptitSeb/box64#954] [Announcements]
This looks very interesting. How does it compare and contrast with Parallels 19's new functionality where it runs x64 Linux binaries via Rosetta on macOS?

**Comment by ptitSeb** [comment]
I'm not sure what kind of answer you expect here? But I convert thsi to a discussion.

**Reply by ylluminate** [reply]
While both Parallels 19 with Rosetta and Box64 aim to enable x86_64 Linux binary execution on ARM64 systems, their approaches differ significantly. 

Parallels uses Rosetta 2, which adds an extra layer of abstraction, while Box64 seems to operate directly on the host OS. 

This could translate to performance differences and perhaps variations in compatibility and feature support... 

Box64 appears to be optimized through "DynaRec" (dynamic recompiler), promising speed enhancements. It would be fascinating to run benchmark tests comparing the two. Do you have any such information?

Or can you share any more direct methods of using box64 on/with macOS that you've seen?

It seems worthwhile to create and consider such benchmarks to evaluate the potential need (or lack thereof) for Parallels 19 and integration of box64 into UTM / qemu-based projects...
